Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  TodosViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 26/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class TodosViewController: UIViewController {
+    // TODO: Replace todos with real data
+    private lazy var todos = setupTodos()
```

Comment:
Hmm, shouldn't this be fetched from the server?

Reply:
Yes it should. I saw that we have just main branch, and assumed it's done, but I need to implement fetching data.

Reply:
Indeed. Lets close this PR and proceed with implementing the data fetch.